Afterschool/Afterschool/Presentation/Common/Components/NavigationBar.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  NavigationBar.swift
+//  Afterschool
+//
+//  Created by 임영택 on 8/25/25.
+//
+
+import SwiftUI
+
+/// 주어진 뷰를 VStack으로 감싸고, 상단에 커스텀 네비게이션 바를 붙이는 뷰 모디파이어
+///
+/// - Parameters:
+///   - centerView: 네비게이션 바의 중앙에 표시할 커스텀 뷰 (예: 텍스트, 로고 등)
+///   - inversed: 네비게이션 바 색상 반전 여부. `true`면 어두운 배경/흰색 아이콘, `false`면 밝은 배경/검정 아이콘
+struct NavigationBar<CenterView>: ViewModifier where CenterView: View {
+    @Environment(\.dismiss) var dismiss
+    
+    /// MARK: - Properties
+    let centerView: CenterView
+    let inversed: Bool
+    
+    /// MARK: - Constraints
+    private let backButtonImageWidth: CGFloat = 15
+    private let backButtonImageHeight: CGFloat = 21
+    private let navBarHeight: CGFloat = 76
+    private let horizontalPadding: CGFloat = 20
+    private let inversedBackgroundColor: Color = Color(hex: "545454")
+    
+    func body(content: Content) -> some View {
+        VStack(spacing: 0) {
+            ZStack {
+                HStack { // 좌측 버튼
+                    Button {
+                        dismiss()
+                    } label: {
+                        Image(systemName: "chevron.left")
+                            .resizable()
+                            .aspectRatio(contentMode: .fit)
+                            .foregroundColor(!inversed ? Color.black : Color.white)
+                            .frame(width: backButtonImageWidth, height: backButtonImageHeight)
+                    }
+                    
+                    Spacer()
+                }
+                .frame(height: navBarHeight)
+                .frame(maxWidth: .infinity)
+                .padding(.horizontal, horizontalPadding)
+                
+                HStack { // 가운데 뷰
+                    Spacer()
+                    centerView
+                        .frame(maxHeight: navBarHeight)
+                    Spacer()
+                }
+            }
+            .background(!inversed ? Color.white : inversedBackgroundColor)
```

Comment:
```suggestion
            .background(!inversed ? Color.afWhite : inversedBackgroundColor)
```

위에서 언급한 것과 같은 제안입니다!

Reply:
좋습니다 반영했습니다!